main.py

```diff
@@ -0,0 +1,58 @@
+from sklearn import tree, model_selection
+import graphviz
+from sklearn import ensemble
+from sklearn import preprocessing
+import xgboost as xgb
+from sklearn.model_selection import train_test_split
+import joblib
+import pandas as pd
+
+filename = 'weights/titanic_survivor_grad_boost_classifier.pkl'
+
+
+def convert_csv_to_arrays(path_to_csv: str, train: bool):
+    original_df = pd.read_csv(path_to_csv, sep=',', header=0)\
```

Comment:
`sep=',', header=0` are default parameters in `pd.read_csv`, you can skip them the next time😉

Reply:
`\` is not a preferable way to break a line. You can easily do this after opening brackets or after comma, e.g.
```suggestion
    original_df = pd.read_csv(path_to_csv, sep=',', 
                              header=0).drop('Ticket', axis=1)
```